src/Compatibility/Microsoft.DotNet.ApiCompat.Task/ValidateAssembliesTask.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+
+namespace Microsoft.DotNet.ApiCompat.Task
+{
+    public class ValidateAssembliesTask : TaskBase
+    {
+        [Required]
+        public string? RoslynAssembliesPath { get; set; }
+
+        [Required]
+        public string[]? LeftAssemblies { get; set; }
```

Comment:
Should we allow null for left, right and RoslynAssemblies sets?

Reply:
RoslynAssembliesPath doesn't need to be provided, it's just a mechanism to load a different roslyn version, hence its null today. LeftAssemblies and RightAssemblies are marked as required, hence they must be provided - msbuild checks for that.

Reply:
>RoslynAssembliesPath doesn't need to be provided

Then why it's marked as Required?